src/emptyState/story.tsx

```diff
@@ -4,7 +4,7 @@ import { storiesOf } from '@storybook/react'
 import { withKnobs, text, boolean } from '@storybook/addon-knobs'
 import spec from 'emptyState/specifications/emptystate.md'
 
-import EmptyState from 'emptyState'
+import EmptyState from './index'
```

Comment:
nit: I think that you don't need to change it.

Reply:
I need to get the new styles component in index.jsx